bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleStatus.java

```diff
@@ -1,4 +1,8 @@
 /**
+ * Copyright (c) 2017 by Bosch Software Innovations GmbH. All rights reserved,
```

Comment:
Well, yes, it is against the agreed rules for the license headers in this project (or rather this component).
If I may repeat them: We use
```
 * Copyright (c) {DATE} {INITIAL COPYRIGHT OWNER} and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
```
where `{INITIAL COPYRIGHT OWNER}` is the party that created the file in the first place (in this case ProSyst) and "and others" includes every one else that has done subsequent changes to it (which imho includes successor companies such as Bosch SI).

We definitely must not have two different license headers on a file.
In any case, please remember that we plan to move to a completely [new style of license headers](https://bugs.eclipse.org/bugs/show_bug.cgi?id=514948) in future.